docs/dev/testing.md

```diff
@@ -38,10 +38,17 @@ If you want to run only a specific test:
 
     $ ./test.py suitename/testname
 
-If you want to run only a specific test-case from a test:
+This will select and run all tests having suitename/testname as substring,
+e.g. suitename/testname_ext and others.
```

Comment:
So in this case, it's not really a substring, it's a prefix? Would `suitename/hello_testname` also match, or not?

@kostja @kbr-scylla and others - please consider if this is what you really wanted to happen. I think if you provide a "full path" of the test - suite and test file - then it should be an exact match, not a substring or prefix match. I think the substring should only be done when just a single word (e.g., "lwt") is given. But maybe I am missing the original intent of this feature?

Reply:
It's a substring, the check is `if patter in testname`, so suite/foo wouldn't match suite/ext_foo, but plain foo would